bld_commands/src/hist/command.rs

```diff
@@ -28,6 +29,11 @@ impl BldCommand for HistCommand {
             .long("server")
             .takes_value(true)
             .help("The name of the server from which to fetch execution history");
+        let state = Arg::with_name(STATE)
+            .short('x')
+            .long("state")
+            .takes_value(true)
+            .help("Filter the pipelines execution history with the provided state.");
         SubCommand::with_name(HIST)
```

Comment:
The state argument is not included in the Subcommand

```suggestion
        SubCommand::with_name(HIST)
            .about("Fetches execution history of pipelines on a server")
            .version(VERSION)
            .args(&[server, state])
```